LICENSE.txt

```diff
@@ -33,58 +33,3 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
-
```

Comment:
are these licenses still needed, or did they move somewhere else?
the project still uses Antlr, not sure why codepoint is included.

Reply:
Looks like these were moved to THIRD_PARTY_LICENSES.txt, so all good.